packages/calling/src/CallHistory/CallHistory.ts

```diff
@@ -202,6 +254,45 @@ export class CallHistory extends Eventing<CallHistoryEventTypes> implements ICal
     }
   }
 
+  /**
+   * Function to display the UCM Lines API response.
+   * @returns {Promise} Resolves to an object of type  {@link UCMLinesResponse}.Response details with success or error status.
+   */
+  public async fetchUCMLinesData(): Promise<UCMLinesResponse> {
```

Comment:
Does this have to be a public function? It's being used within getCallHistoryData() right? Is there going to be any scenario where developers will directly need to call this method?

Reply:
Taken care of it. We've changed fetchUCMLinesData() from public to private since it's only being used within getCallHistoryData().